src/types.ts

```diff
@@ -46,49 +47,57 @@ export interface BallotPageMetadata {
 
 export interface BallotPageContestLayout {
   bounds: Rect
+  corners: Corners
   options: readonly BallotPageContestOptionLayout[]
 }
 
 export interface BallotPageContestOptionLayout {
   bounds: Rect
-  target: Rect
+  target: TargetShape
```

Comment:
The changes in this file, such as this one, constitute a breaking change to the `Interpreter` API depending on how it was used.